qc_tests/streaks.py

```diff
@@ -4,28 +4,33 @@
 
    Checks for replication of 
      1. checks for consecutive repeating values
-     2. checks if one year has more repeating strings than expected [Not yet implemented]
+     2. checks if one year has more repeating streaks than expected
      3. checks for repeats at a given hour across a number of days [Not yet implemented]
-     4. checks for repeats for whole days - all 24 hourly values [Not yet implemented]
+     4. checks for repeats for whole days - all 24 hourly values
```

Comment:
Double check this explanation tallies up with the QC tests/functions within. It would be easier to follow if you add the relevant function names to the docstrings here I think. I'm not entirely clear what tests are done here or where the humidity variables only have persistent 0 DPD tested or whether that is in addition?